cli/build.gradle.kts

```diff
@@ -26,17 +27,24 @@ dependencies {
     runtimeOnly("org.apache.maven:maven-artifact:${project.extra.get("mavenVersion")}")
 
     testRuntimeOnly("commons-io:commons-io:${project.extra.get("commonsVersion")}")
-    testImplementation(testFixtures(project(":common")))
+    //testImplementation(testFixtures(project(":common")))
+    testImplementation(project(path = ":common", configuration = "testFixturesCompile"))
+
     testImplementation(project(":analyzer"))
     testImplementation("junit:junit:${project.extra.get("junitVersion")}")
+    testImplementation("org.assertj:assertj-core:${project.extra.get("assertjVersion")}")
     testImplementation("com.github.stefanbirkner:system-rules:${project.extra.get("systemRulesVersion")}")
     testImplementation("org.codehaus.plexus:plexus-archiver:4.2.3")
 }
 
 tasks {
     "jar"(Jar::class) {
-        this.manifest {
+        manifest {
             attributes["Main-Class"] = "org.jboss.gm.cli.Main"
         }
     }
+
+    "test" {
+        dependsOn(":analyzer:shadowJar")
+    }
```

Comment:
Why this change please?

Reply:
This actually came from a complaint from Gradle 7, I believe. We could remove it until that support is complete. It complains that the `cli` depends on the `shadowJar` task, but doesn't declare it anywhere. There is `testImplementation(project(":analyzer"))` but that doesn't seem to be doing it.

Reply:
Ok ; see https://github.com/project-ncl/gradle-manipulator/pull/319 which changes it slightly.